Tensile/Source/lib/include/Tensile/Serialization/PlaceholderLibrary.hpp

```diff
@@ -68,7 +74,11 @@ namespace Tensile
                     for(auto condition : ctx->preloaded)
                     {
                         std::string pattern = RegexPattern(condition);
-                        if(std::regex_search(lib.filePrefix, std::regex(pattern)))
+#ifdef WIN32
+                        if(PathMatchSpec(lib.filePrefix.c_str(), pattern.c_str()))
+#else
```

Comment:
Double check if tensile is using specific char set form to avoid runtime errors if build type changes, i.e. PathMatchSpecA may be preferred.   Also appears shlwapi usage varies a lot across rocm, delay load dll to static and other forms of use may cause deployment runtime errors.  See if you can find out from other libraries before adding this new dependency.

Reply:
shlwapi is used similar to how it is linked in this PR across other libraries. onnxrt uses delayload, but I do not think it might be needed in this case. I have compiled rocblas on windows with this change and it compiles fine and the pattern matching works as expected.